dask/utils_test.py

```diff
@@ -123,3 +129,16 @@ def import_or_none(name):
         return importlib.import_module(name)
     except (ImportError, AttributeError):
         return None
+
+
+def hlg_layer(hlg: HighLevelGraph, prefix: str) -> Layer:
+    "Get the first layer from a HighLevelGraph whose name starts with a prefix"
+    for key, lyr in hlg.layers.items():
+        if key.startswith(prefix):
+            return lyr
+    raise KeyError(f"No layer starts with {prefix!r}: {list(hlg.layers)}")
+
+
+def hlg_layer_topological(hlg: HighLevelGraph, i: int) -> Layer:
```

Comment:
I'm wondering if it makes sense to just return the `i=-1` layer since It looks like we're always calling this with `i=-1` (outside of `test_hlg_layer_topological`)

Reply:
I thought about that. It would probably be nice if `i=-1` was the default, since I think that'll be the most common use. I just couldn't think of a name that made it obvious it returned the last layer by default, but where that could also be changed (`last_hlg_layer(hlg, i=-4)` sounds weird when `i` isn't -1). I just wanted someone reading a test to not have to go look at the implementation for this function to know what the default was. So since I couldn't come up with a good name, I just didn't give a default.